test/workload-deployment.sh

```diff
@@ -0,0 +1,140 @@
+
+set -e # exit on error
+
+echo "Install KubeFlex"
+echo "-------------------------------------------------------------------------"
+OS_ARCH=linux_amd64
+LATEST_RELEASE_URL=$(curl -H "Accept: application/vnd.github.v3+json"   https://api.github.com/repos/kubestellar/kubeflex/releases/latest   | jq -r '.assets[] | select(.name | test("'${OS_ARCH}'")) | .browser_download_url')
+curl -LO $LATEST_RELEASE_URL
+mkdir -p kubeflex
+tar xzvf $(basename $LATEST_RELEASE_URL) -C kubeflex
+rm $(basename $LATEST_RELEASE_URL)
+export PATH=$PWD/kubeflex/bin:$PATH
+
+echo "Create a Kind hosting cluster with nginx ingress controller and KubeFlex operator"
+echo "-------------------------------------------------------------------------"
+kflex init --create-kind
+
+echo "Create an inventory & mailbox space of type vcluster running OCM (Open Cluster Management) directly in KubeFlex. Note that -p ocm runs a post-create hook on the vcluster control plane which installs OCM on it."
+echo "-------------------------------------------------------------------------"
+kflex create imbs1 --type vcluster -p ocm
+
+echo "Create a Workload Description Space wds1 directly in KubeFlex. Similarly to before, -p kslight runs a post-create hook on the k8s control plane that starts an instance of a KSLight controller manager which connects to the wds1 front-end and the imbs1 OCM control plane back-end."
+echo "-------------------------------------------------------------------------"
+kflex create wds1 -p kslight
+
+echo "Register clusters with OCM"
+echo "-------------------------------------------------------------------------"
+./register_cluster1_cluster2_with_ocm.sh
+
+echo "Check all deployments and statefulsets running in the hosting cluster"
+echo "-------------------------------------------------------------------------"
+kubectl --context kind-kubeflex get deployments,statefulsets --all-namespaces
+
+echo "Check for available clusters with label location-group=edge"
+echo "-------------------------------------------------------------------------"
+kubectl --context imbs1 get managedclusters -l location-group=edge
+
+echo "Create a placement to deliver an app to all clusters in wds1."
+echo "This placement configuration determines where to deploy the workload by using the label selector expressions found in clusterSelectors. It also specifies what to deploy through the downsync.labelSelectors expressions. When there are multiple matchLabels expressions, they are combined using a logical AND operation. Conversely, when there are multiple labelSelectors, they are combined using a logical OR operation."
+echo "-------------------------------------------------------------------------"
+kubectl --context wds1 apply -f - <<EOF
+apiVersion: edge.kubestellar.io/v1alpha1
+kind: Placement
+metadata:
+  name: nginx-placement
+spec:
+  clusterSelectors:
+  - matchLabels: {"location-group":"edge"}
+  downsync:
+  - labelSelectors:
```

Comment:
We have this changed to `objectSelectors`, no? (#1546)

Reply:
Good point.  But 1546 isn't merged yet.

Reply:
Maybe I should mention #1525 as well, APIs changes are merged there and 1546 is based on those changes.

Reply:
There is currently a long and undocumented path from the source to what this test runs.

Reply:
I rebased and changed labelSelectors to objectSelectors.